.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "<app name>: <description>"
+labels: kind/bug, triage/support
+assignees: ''
+
+---
```

Comment:
Is this for use? Does it show once the template is used and the issue is created?
Do you want to use comments instead?

Should labels be automatic instead?